projects/vdk-control-cli/tests/vdk/internal/control/command_groups/info_group/test_info.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from typing import Any
+from typing import List
+
+from click.testing import CliRunner
+from py._path.local import LocalPath
+from pytest_httpserver.pytest_plugin import PluginHTTPServer
+from vdk.internal import test_utils
+from vdk.internal.control.command_groups.info_group.info import info
+from vdk.internal.test_utils import assert_click_status
+
+test_utils.disable_vdk_authentication()
+
+
+def get_data_job_query_response(jobs: List[Any]):
+    return dict(data=dict(content=jobs, errors=[]))
+
+
+def get_example_info_response():
+    return {
+        "api_version": "PipelinesControlService/1.6.855638094/v0.13-30-g23b0851 (root@vdk-cs-dep-6f88c4bff-rzbbl Linux/amd64/amd64)",
+        "supported_python_versions": ["3.7", "3.8", "3.9"],
```

Comment:
in the api.yaml file the example responses has all responses prefixed with `python`

Reply:
Where is the fix? I don't see it in this PR.